setup.py

```diff
@@ -147,7 +147,7 @@ def package_files(directory):
         "scipy<1.9.2",
         "sqlalchemy",
         "uvicorn >= 0.17.0",
-        "websockets >= 9.0.1, <11.0",
+        "websockets >= 9.0.1",
```

Comment:
9.0.1 is from may 2021, maybe old enough that we can drop it as a lower bound (we probably don't even know if that lower bound is sufficient)